types/cucumber/index.d.ts

```diff
@@ -62,14 +62,15 @@ declare namespace cucumber {
     }
 
     interface HookOptions{
-  		timeout?: number;
-  	}
+	timeout?: number;
+	tags?: any;    
+    }
 
     export interface Hooks {
         Before(code: HookCode): void;
         Before(options: HookOptions, code: HookCode): void;
-		    After(code: HookCode): void;
-		    After(options: HookOptions, code: HookCode): void;
+	After(code: HookCode): void;
```

Comment:
If you open the link, you will see it is proper set https://github.com/manu7691/DefinitelyTyped/blob/1f55084159d14af7cf93cff2e758714d0ff2b7cf/types/cucumber/index.d.ts